openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -2020,7 +2020,7 @@ def createForce(self, system, topology, nonbondedMethod=NoCutoff, nonbondedCutof
             for atom_mapping in atom_mappings:
                 for (atom_index, map_atom_index) in atom_mapping.items():
                     # Retrieve NB params for reference atom (charge not set yet)
-                    [charge, sigma, epsilon] = force.getParticleParameters(atom_index)
+                    charge, sigma, epsilon = force.getParticleParameters(map_atom_index)
```

Comment:
It's not clear what the difference between `atom_index` and `map_atom_index` is, perhaps we should put a comment above the for loop? I'm not particularly familiar with this portion of the code though so maybe it isn't necessary. @davidlmobley